packages/space-opera/src/components/model_viewer_snippet/model_viewer_snippet.ts

```diff
@@ -121,16 +128,20 @@ export class ExportPanel extends ConnectedLitElement {
     <snippet-viewer id="snippet-header" .renderedSnippet=${snippet}
       .renderedStyle=${this.hotspots.length > 0 ? hotspotStyles.cssText : ``}>
     </snippet-viewer>
+    <mwc-button unelevated
```

Comment:
moved edit snippet button to be under export snippet on first tab (seen in image from prev comment)